pkg/util/arangod/client.go

```diff
@@ -65,23 +65,23 @@ var (
 		Proxy: nhttp.ProxyFromEnvironment,
 		DialContext: (&net.Dialer{
 			Timeout:   30 * time.Second,
-			KeepAlive: 30 * time.Second,
+			KeepAlive: 100 * time.Millisecond,
 			DualStack: true,
 		}).DialContext,
 		MaxIdleConns:          100,
-		IdleConnTimeout:       90 * time.Second,
+		IdleConnTimeout:       100 * time.Millisecond,
```

Comment:
These are shared instances, so you're now changings this for ALL. That is not good.